q2_feature_classifier/_cutter.py

```diff
@@ -36,6 +36,8 @@ def _primers_to_regex(f_primer, r_primer):
 def _local_aln(primer, sequence):
     best_score = None
     for one_primer in primer.expand_degenerates():
+        # `sequence` may contain degenerates. Although undocumented, SSW will
+        # treat each degenerate character as a mismatch.
```

Comment:
Ok, in that case, I'm not sure that that's quite correct. The only degenerates in `sequence` should be `N` characters, and from memory SSW treats them specially, allocating them a score of zero.